src/index.html

```diff
@@ -17,16 +17,28 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar big-search-bar"
+      data-qa="big"
+    >
+      <input
+        class="search-bar-input search-bar-input--big"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar small-search-bar"
+      data-qa="small"
```

Comment:
Class names should represent the meaning of the content, not the styles. The class name 'small-search-bar' may not be the best choice. Consider using a name that describes the function or content of the element, such as 'secondary-search-bar' or 'auxiliary-search-bar'.